app/reducers/taps.js

```diff
@@ -492,6 +506,13 @@ export default function taps(state = defaultState, action) {
             app_secret: '',
             start_date: ''
           }
+        },
+        'tap-s3-csv': {
+          fieldValues: {
+            bucket: '',
+            start_date: '',
+            tables: ''
```

Comment:
Default to `[]` instead of `''`?

Reply:
Converting to a string before saving to store since the config expects it to be a sting.